src/high-level/include/iDynTree/KinDynComputations.h

```diff
@@ -371,13 +486,34 @@ class KinDynComputations {
      */
     iDynTree::Transform getWorldTransform(const iDynTree::FrameIndex frameIndex);
 
+    /**
+     * Return the transform where the frame is the frame
+     * specified by frameIndex, and the reference frame is the world one
+     * (world_T_frame).
+     * @param world_T_frame
+     * @warning the Span and the MatrixView objects should point an already existing memory. Memory allocation and resizing cannot be achieved with this kind of objects.
+     * @return true if all went well, false otherwise.
```

Comment:
also here. Please check all the doc

Reply:
See https://github.com/robotology/idyntree/pull/736#discussion_r488465087